### DISCUSSION TITLE
[Feature] Add ability to store a group of OTP keys on a folder

### DISCUSSION THREAD

**Discussion by rihards-simanovics** [2fast-team/2fast#38] [Ideas]
## Describe the problem this feature would solve
It would be nice to have an ability to sort the keys by the "folders" so for instance all work related OTP in "Work" folder, all personal keys in the "Personal" folder, etc.

## Describe the solution
add option to have folders

## Describe alternatives you've considered
hmm perhaps filters or tags (though I like this less)?

## Additional context & Screenshots
Perhaps something similar to what Files (Open-Source alternative to File Explorer) does:
![image](https://user-images.githubusercontent.com/52106095/203506883-ea68d35a-4380-450f-992f-de76f2d32c89.png)

**Comment by jp-weber** [comment]
@rihards-simanovics 

This is an interesting idea and would split the current view into two modes. At the moment I only planned to be able to set categories and filter by category in the future, but this would be another option to map categories.
